development/decision_process.rst

```diff
@@ -0,0 +1,182 @@
+######################################
+Discussion and Decision Making Process
+######################################
+
+This page describes how decisions are made in Data Management.
+Generally, DM teams and staff have agency to implement non-controversial work (see § :ref:`decision-making-empowerment`).
+
+For cases where you have a well-formed proposal that requires project consensus we use a :ref:`Request for Comments (RFC) <decision-making-rfc>` process.
+
+You may also want to convene members across DM for an in-depth technical discussion.
+The :ref:`Request for Discussion (RFD) <decision-making-rfd>` process facilitates DM-wide discussions in a weekly discussion time slot.
+
+.. _decision-making-empowerment:
+
+Empowerment
+===========
+
+You are empowered by the DM Project Manager (PM, Jeff Kantor) and Project Scientist (PS, Mario Juric) to make decisions on any DM-internal matter---such as technical/algorithm issues, process improvements, and tool choices---when:
+
+- you are willing and able to do the work to implement the decision yourself or with people who agree with you,
+- you (collectively) are willing and able to fix any problems if it goes wrong, and
+- you believe that all affected parties (including your immediate manager) would not seriously object to your decision and implementation.
+
+If the these three criteria are not met, perhaps because you don't know all the affected parties or because you don't know their positions, you should publish your proposed decision and implementation as a JIRA RFC issue to seek awareness and consensus.
```

Comment:
Also, "JIRA RFC issue" seems less clear to me than "JIRA issue in the RFC project".  Since we now have RFD issues, we might need to say "JIRA issue of type RFC in the RFC project" despite the seeming redundancy.